ui/app/models/secret-engine.js

```diff
@@ -13,6 +13,18 @@ const Validations = buildValidations({
     presence: true,
     message: "Path can't be blank.",
   }),
+  maxVersions: [
```

Comment:
used for the creation of a new secret engine now that we have config data that needs to validation maxVersion.